src/codewhisperer/service/inlineCompletion.ts

```diff
@@ -401,6 +405,21 @@ export class InlineCompletion {
                     this.setRange(new vscode.Range(editor.selection.active, editor.selection.active))
                     try {
                         await this.showRecommendation(editor)
+                        let languageId = editor?.document?.languageId
+                        languageId =
+                            languageId === CodeWhispererConstants.typescript
+                                ? CodeWhispererConstants.javascript
+                                : languageId
+                        const languageContext = runtimeLanguageContext.getLanguageContext(languageId)
```

Comment:
This will be conflicting with https://github.com/aws/aws-toolkit-vscode/pull/2859

Reply:
this can go first, I will merge and resolve conflicts thereafter

Reply:
thank you @Will-ShaoHua. I can rebase on yours since yours is already good to merge?